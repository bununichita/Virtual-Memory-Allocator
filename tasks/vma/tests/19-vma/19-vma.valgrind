==4247== Memcheck, a memory error detector
==4247== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==4247== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==4247== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==4247== Parent PID: 4246
==4247== 
==4247== 
==4247== HEAP SUMMARY:
==4247==     in use at exit: 0 bytes in 0 blocks
==4247==   total heap usage: 28 allocs, 28 frees, 8,765 bytes allocated
==4247== 
==4247== All heap blocks were freed -- no leaks are possible
==4247== 
==4247== For lists of detected and suppressed errors, rerun with: -s
==4247== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
