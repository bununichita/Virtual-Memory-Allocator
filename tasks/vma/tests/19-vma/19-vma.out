Total memory: 14 bytes
Free memory: 16 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 4

Block 1 begin
Zone: 3 - 7
Miniblock 1:	3	-	4	| RW-
Miniblock 2:	4	-	5	| RW-
Miniblock 3:	5	-	6	| RW-
Miniblock 4:	6	-	7	| RW-
Block 1 end
Total memory: 14 bytes
Free memory: 17 bytes
Number of allocated blocks: 2
Number of allocated miniblocks: 3

Block 1 begin
Zone: 3 - 4
Miniblock 1:	3	-	4	| RW-
Block 1 end

Block 2 begin
Zone: 5 - 7
Miniblock 1:	5	-	6	| RW-
Miniblock 2:	6	-	7	| RW-
Block 2 end
Total memory: 14 bytes
Free memory: 16 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 4

Block 1 begin
Zone: 3 - 7
Miniblock 1:	3	-	4	| RW-
Miniblock 2:	4	-	5	| RW-
Miniblock 3:	5	-	6	| RW-
Miniblock 4:	6	-	7	| RW-
Block 1 end
Total memory: 14 bytes
Free memory: 17 bytes
Number of allocated blocks: 2
Number of allocated miniblocks: 3

Block 1 begin
Zone: 3 - 5
Miniblock 1:	3	-	4	| RW-
Miniblock 2:	4	-	5	| RW-
Block 1 end

Block 2 begin
Zone: 6 - 7
Miniblock 1:	6	-	7	| RW-
Block 2 end
2
3
4
1
6
